app/views/subscriptions/_estimate_notifications_modal.html.erb

```diff
@@ -5,6 +5,7 @@
     <p class="status-text"></p>
   </div>
   <div class="results-text">
+    <p class="query-text"></p>
```

Comment:
This can be 40k characters long; I think an option to collapse the text would be a good inclusion.

Reply:
Does MMT have a UX model for the collapse?

Reply:
We use eui accordions in the forms. I'm still thinking if there's some place we use it on modals